src/components/SearchForm/SearchForm.js

```diff
@@ -1,15 +1,37 @@
 import React from "react";
 import "../SearchForm/SearchForm.css";
 
-function SearchForm() {
+function SearchForm({ onSubmit }) {
+
+    const [searchWord, setSearchWord] = React.useState("");
+    const [placeholder, setPlaceholder] = React.useState("Enter topic");
+
+    function handleInput(e) {
+        setSearchWord(e.target.value);
```

Comment:
**Could be improved**

You can make a custom hook for inputs handling:

```js
export function useForm(inputValues) {
  const [values, setValues] = useState(inputValues);

  const handleChange = (event) => {
    const {value, name} = event.target;
    setValues({...values, [name]: value});
  };
  return {values, handleChange, setValues};
}

```

This way you can easily handle any form in your project
You should put it in a separate file  `useForm.js` in folder `hooks` and then import it to the component that needs form handling:

And now only one line of code enables form handling:

```jsx

const {values, handleChange, setValues} = useForm({})


```